.github/workflows/hosted-notebook-samples-check.yml

```diff
@@ -0,0 +1,19 @@
+name: Hosted Notebooks Samples Check
+
+on:
+  pull_request:
+    branches: 
+      - feature/samples-gallery
+
+jobs:
+  build:
+    name: Call Azure Pipeline
+    runs-on: ubuntu-latest
+    steps:
+      - name: Azure Pipelines Action
+        uses: Azure/pipelines@v1
+        with:
+          azure-devops-project-url: ${{ secrets.QUANTUM_PROGRAM_ADO_URL }}
+          azure-pipeline-name:  ${{ secrets.HOSTED_NOTEBOOKS_PIPELINE_NAME }}
+          azure-devops-token: ${{ secrets.HOSTED_NOTEBOOKS_PIPELINE_ADO_PAT }}
+          azure-pipeline-variables: '{"Quantum Computing Samples Commit Hash": "$GITHUB_SHA"}'
```

Comment:
Minor, but this pipeline variable name feels a bit long to me, making it somewhat prone to mis-type?